Update all hardhat plugin pages (#412)
matter-labs/zksync-web-era-docs: docs/api/hardhat/hardhat-zksync-vyper.md

::: warning

- Compilers are no longer released as Docker images and its usage is no longer recommended. 
- Use the `compilerSource: "binary"` in the `hardhat.config.ts` file to use the binary instead.

:::

- `version` is the `zkvyper` compiler version. Compiler versions can be found in [the following repository](https://github.com/matter-labs/zkvyper-bin).
- `compilerSource` indicates the compiler source and can be either `docker` or `binary`(recommended). If there isn't a compiler binary already installed, the plugin will automatically download it. If `docker` is used, you would need to run Docker desktop in the background and provide both `dockerImage` and `tag` in the experimental section.

> - `compilerPath` is a field with the path to the `zkvyper` binary. By default, the binary in `$PATH` is used. If `compilerSource` is `docker`, this field is ignored, this filed is optional.

Reviewer: ```suggestion
- `compilerPath` is an optional field with the path to the `zkvyper` binary. By default, the binary in `$PATH` is used. If `compilerSource` is `docker`, this field is ignored.
```
